AirCasting/SessionViews/ABMeasurementsView.swift

```diff
@@ -6,11 +6,19 @@ enum MeasurementPresentationStyle {
     case hideValues
 }
 
+class ABMeasurementsViewThreshold: ObservableObject {
```

Comment:
Why this class was needed instead of changing view properties to a `@State`? (applies to the `SingleMeasurementViewThreshold` too)